internal/pkg/artifact/config.go

```diff
@@ -74,17 +111,19 @@ func (r *Reloader) Reload(rawConfig *config.Config) error {
 	}
 	cfg := &reloadConfig{}
 	if err := rawConfig.Unpack(&cfg); err != nil {
-		return errors.New(err, "failed to unpack config during reload")
+		return err
```

Comment:
what is the benefit of removing the additional context from the the error here? it was wrapping the error before the change